ui/app/utils/clickhouse/inference.ts

```diff
@@ -162,8 +165,12 @@ export const episodeByIdSchema = z
   .object({
     episode_id: z.string().uuid(),
     count: z.number().min(1),
-    start_time: z.string().datetime(),
-    end_time: z.string().datetime(),
+    start_time: z
+      .string()
+      .transform((str) => new Date(`${str.replace(" ", "T")}Z`)),
+    end_time: z
+      .string()
+      .transform((str) => new Date(`${str.replace(" ", "T")}Z`)),
```

Comment:
If we do `formatDateTime(max(timestamp), '%Y-%m-%dT%H:%i:%SZ') AS max_timestamp,` for all timestamps in the ClickHouse query then we can use the simple `.string().datetime()` type in Zod which I think is cleaner. Let's move this back and fix all datetimes in the queries that way.